RiotSharp.AspNetCore/IServiceCollectionExtension.cs

```diff
@@ -33,6 +33,8 @@ public static IServiceCollection AddRiotSharp(this IServiceCollection serviceCol
             {
                 var rateLimitedRequester = new RateLimitedRequester(riotSharpOptions.RiotApi.ApiKey,
                     riotSharpOptions.RiotApi.RateLimits);
+                var nonLimitingRequester = new Requester(riotSharpOptions.RiotApi.ApiKey);
```

Comment:
why do you need it?

Reply:
Someone added the status endpoint to the API, which lead to the API taking another rate limiter in it's constructor.

IDK if its really necessary, but if the status API is not limited, reusing the rate limited requester would lead to rate limits potentially being fired without hitting a limited endpoint.

Reply:
The problem here is: Right now every requester creates his own `HttpClient` instance. However, the `HttpClient` is threadsafe and designed for concurrent requests, therefore we should reuse the same instance for every requester.  When I migrated the status API, I intentionally left the build error there to fix it properly in a later commit.

Reply:
@JanOuborny Alright, removed that part from the PR :)